artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/impl/ClusterConnectionImpl.java

```diff
@@ -801,7 +801,6 @@ private void createNewRecord(final long eventUID,
       targetLocator.setInitialConnectAttempts(0);
       targetLocator.setClientFailureCheckPeriod(clientFailureCheckPeriod);
       targetLocator.setConnectionTTL(connectionTTL);
-      targetLocator.setInitialConnectAttempts(0);
```

Comment:
what is the outcome of this? faster ? failure? or just to be correct?

Reply:
`setInitialConnectAttempts` has been set to 0 two lines before, no need to set it again.

Reply:
OIC